partiql-ast/src/ast/mod.rs

```diff
@@ -647,12 +646,6 @@ pub struct List {
     pub values: Vec<Box<Expr>>,
 }
 
-#[derive(Visit, Clone, Debug, PartialEq)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct Sexp {
-    pub values: Vec<Box<Expr>>,
-}
```

Comment:
also need a changelog update for public API removal

Reply:
Yep. I was preparing it presently. I didn't expect you to get to the PR so fast. :)